app/modifiers/is-dark-mode.ts

```diff
@@ -1,56 +1,139 @@
-// Invokes a callback when the dark mode setting changes (and on initial load).
-// Usage: <div {{is-dark-mode this.handleDarkModeUpdated}}></div>
 import { registerDestructor } from '@ember/destroyable';
-import { action } from '@ember/object';
-import { task } from 'ember-concurrency';
+import { service } from '@ember/service';
+import { task, timeout } from 'ember-concurrency';
 import Modifier from 'ember-modifier';
+import type DarkModeService from 'codecrafters-frontend/services/dark-mode';
 
+/**
+ * Enable or disable the DOM Mutation Observer
+ */
+const DOM_MUTATION_OBSERVER_ENABLED: boolean = false; // TODO: Figure out an efficient way to enable the observer while avoiding too many calls
+
+/**
+ * Enable or disable subscribing to `DarkModeService.isEnabled` changes
+ */
+const DARK_MODE_UPDATE_LISTENER_ENABLED: boolean = true;
```

Comment:
Avoid configuration options as far as possible, I'd suggest just removing this and keep it enabled

Reply:
Removed this helper completely